posts/2021-05-11-database-ssl-primer.adoc

```diff
@@ -183,93 +191,98 @@ Property Documentation: https://jdbc.postgresql.org/documentation/head/connect.h
     <variable name="oracle.keystore" value="javax.net.ssl.keystore=security/keystore.p12;javax.net.ssl.trustStoreType=PCKS12;javax.net.ssl.keyStorePassword=${KEYSTORE_PASSWORD};"/>
     <variable name="oracle.truststore" value="javax.net.ssl.trustStore=security/truststore.p12;javax.net.ssl.trustStoreType=PCKS12;javax.net.ssl.trustStorePassword=${TRUSTSTORE_PASS};"/>
     <variable name="oracle.conn.props.store" value="${oracle.tls.props}${oracle.truststore}${oracle.keystore}" />
-    
+
     <dataSource jndiName="jdbc/oracle-tls-truststore" jdbcDriverRef="jdbcLib">
     	<properties.oracle URL="${env.SSL_URL}" connectionProperties="${oracle.conn.props.store}" />
     </dataSource>
 
 </server>
 ----
 
-**Enable TLS**
+- Enable TLS
+
+To enable TLS connections on the Oracle JDBC driver, add the `connectionProperties="oracle.net.ssl_version=1.2;"` attribute to the `dataSource properties` element.
+
+The `connectionProperties` attribute takes a semicolon-delimited list of properties to be applied to the driver.
```

Comment:
In the example, `connectionProperties` does not take a semicolon delimited list, that list is in `variable name="oracle.tls.props"`, as are the properties discussed in the following list. This needs clarification because it makes the example difficult to follow. I didn't want to attempt an edit as Im not sure if the problem is in the example or the description.